### ISSUE TITLE
TypeError: can only concatenate str (not "list") to str on Windows

### ISSUE THREAD

**Issue by SeanDS** [gwpy/gwpy#1164]
A Windows user in my group installed gwpy using `conda install -c conda-forge gwpy` (using Python 3, and the latest version of gwpy available on conda-forge as of this post). When they try to import it within a Python terminal though, they get this error:

```
>>> import gwpy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\juvwra\Anaconda3\lib\site-packages\gwpy\__init__.py", line 35, in <module>
    from . import (
  File "C:\Users\juvwra\Anaconda3\lib\site-packages\gwpy\plot\__init__.py", line 29, in <module>
    from . import (
  File "C:\Users\juvwra\Anaconda3\lib\site-packages\gwpy\plot\rc.py", line 79, in <module>
    rcParams.get('text.latex.preamble', []) + tex.MACROS),
TypeError: can only concatenate str (not "list") to str
```

**Comment by SeanDS** [comment]
Looks related to #1099?

**Comment by duncanmmacleod** [comment]
Yes, this was already resolved by #1101, and will be part of the next release, which I am desperately trying to finish off...

**Comment by SeanDS** [comment]
@duncanmmacleod not quite related to this ticket: I see you've pushed the v1.0.0 branch to GitHub (thanks!) but not v0.16.0. That means [this link](https://github.com/gwpy/gwpy/compare/v0.16.0..v1.0.0) on [your release page](https://github.com/gwpy/gwpy/releases/tag/v1.0.0) does not work (at least for me).

**Comment by duncanmmacleod** [comment]
Thanks @SeanDS, I've corrected that to compare 0.15.0 and 1.0.0
